app/assets/stylesheets/course/forum/posts.scss

```diff
@@ -0,0 +1,5 @@
+.course-forum-posts {
+  &.reply {
+    @include forum-post;
```

Comment:
... maybe I add it. I think it makes sense to see the post when you're replying to it...